boltdesignsystem/bolt
# Fixed a markup error on the teaser component

## Summary

Fixed a markup error on the teaser twig

## Details

(Explain the changes in enough detail fo reviewers to understand. Raise any questions for reviewers to consider.)

## How to test

Run the branch locally and confirm that the teaser on `/pattern-lab/?p=components-teaser-status-and-actions` is rendering properly
